src/main/java/com/contentful/java/cma/ModuleContentTypes.java

```diff
@@ -167,15 +164,24 @@ public CMAContentType unPublish(CMAContentType contentType) {
   public CMAContentType update(CMAContentType contentType) {
     assertNotNull(contentType, "contentType");
     assertNotNull(contentType.getName(), "contentType.name");
+
     String contentTypeId = getResourceIdOrThrow(contentType, "contentType");
     String spaceId = getSpaceIdOrThrow(contentType, "contentType");
     Integer version = getVersionOrThrow(contentType, "update");
-    return service.update(
-        version,
-        spaceId,
-        contentTypeId,
-        contentType
-    ).toBlocking().first();
+
+    final CMASystem system = contentType.getSystem();
+    contentType.setSystem(null);
+
+    try {
+      return service.update(
+          version,
+          spaceId,
+          contentTypeId,
+          contentType
+      ).toBlocking().first();
+    } finally {
```

Comment:
Also related to the nulling above, why do it just to undo it in the error case here? Wouldn't it be better to only change sys when the update actually succeeds?